.gitignore

```diff
@@ -20,4 +20,4 @@ docs/_modules
 .coverage
 .vscode/
 **/pip-wheel-metadata
-
+.ropeproject/
```

Comment:
Q: What is this? :D

Reply:
It's generated by [rope](https://pypi.org/project/rope/0.5m4/). I have it in my PyCharm often...